executor/executor.cc

```diff
@@ -221,7 +224,8 @@ struct thread_t {
 static thread_t threads[kMaxThreads];
 static thread_t* last_scheduled;
 // Threads use this variable to access information about themselves.
-static __thread struct thread_t* current_thread;
+//static __thread struct thread_t* current_thread;
+static thread_t* current_thread;
```

Comment:
This is just a temporary change. This was implemented recently and haven't looked into why this was made. Will be making changes as soon as I understand what it is.